requirements-test.txt

```diff
@@ -1,24 +1,33 @@
 certifi==2016.8.31 \
-#    --hash=sha256:f7708a42d86f29ccc7c8c4ff9d34a8d854d8d78eb2973d1f28406bb43d6b8919
+    --hash=sha256:172844a3e2896c924e6aa2657b2e56586a724f27edd0e9592be9353f3a3f6c73
```

Comment:
I agree that it's good to verify hashes in local development if we can - and I think we are for anyone using Docker, because it uses pip to install dependencies, and doesn't every install the Balrog package.